src/MoBi.Assets/AppConstants.cs

```diff
@@ -1256,7 +1256,7 @@ public static class Captions
          public static readonly string WarningsCaption = "Solver Warnings";
          public static readonly string HistoryBrowser = "History";
          public static readonly string NameInUse = "New Name";
-         public static readonly string ReanameWizardCaption = "Rename also";
```

Comment:
I am also fixing all the typos that resharpers shows me... There are A LOT in MoBi